vendor/modules.txt

```diff
@@ -150,7 +150,7 @@ github.com/imdario/mergo
 github.com/jensneuse/abstractlogger
 # github.com/jensneuse/byte-template v0.0.0-20200214152254-4f3cf06e5c68
 github.com/jensneuse/byte-template
-# github.com/jensneuse/graphql-go-tools v0.0.0-00010101000000-000000000000 => github.com/TykTechnologies/graphql-go-tools v1.6.2-0.20200918115423-1f1792f5b6d9
+# github.com/jensneuse/graphql-go-tools v0.0.0-00010101000000-000000000000 => github.com/TykTechnologies/graphql-go-tools v1.6.2-0.20200731074614-80c67fc17e8e
```

Comment:
@gernest could you revert this? it is an incorrect version
and go 1.14 will check consistency between go.mod and modules.txt

Reply:
@spetrunin fixed  9cb38999b3277661f063dc16846347b6f570ef9a